go-controller/.golangci.yml

```diff
@@ -26,3 +28,7 @@ linters-settings:
     disable:
       - fieldalignment
       - shadow
+  revive:
+    rules:
+      # TODO: enable recommended (default) revive rules
```

Comment:
I assume by default its a ALL of things and it would mean a lot of cleanup time because the --fix doesn't help fix all?

Reply:
Yup, it involves a lot of style issues, like renaming variables, etc. Definitely a job for a different PR.